helm/cluster-openstack/values.yaml

```diff
@@ -46,9 +46,20 @@ controlPlane:
     imageRepository: "giantswarm"
     imageTag: 3.5.4-0-k8s
 
-nodeClasses: []  # Class definitions for worker node pools. In addition to a required "name", may include "diskSize", "flavor", "image", and "bootFromVolume" as defined for control plane and bastion above.
+nodeClasses: {}  # Class definitions for worker node pools. The "name" of the class is the key of the object. The properties may include "diskSize", "flavor", "image", and "bootFromVolume" as defined for control plane and bastion above. Example:
+  # default:
+  #   bootFromVolume: true
+  #   diskSize: 50
+  #   flavor: n1.medium
+  #   image: ubuntu-2004-kube-v1.22.9
+  #   name: default
 
-nodePools: []  # Node pools that should exist for the cluster. Each node pool must refer to a class defined in nodeClasses above.
+nodePools: {}  # Node pools that should exist for the cluster. Each node pool must refer to a class defined in nodeClasses above. Example:
+  # lon-3:
+  #   class: default
+  #   name: lon-3
```

Comment:
Maybe worth cleaning this up ?

Reply:
is an example, do you prefer not having it?

Reply:
I think he did mean `name` only.